docs/dev/submissionReview.md

```diff
@@ -0,0 +1,40 @@
+
+### Approving an author to submit to a particular add-on ID for the first time
+When a GitHub user submits a particular add-on for the first time, the submission is blocked pending initial review from NV Access.
+This is to confirm that the GitHub user has authorisation to submit the add-on from the add-on maintainers.
+In some cases, 2 separate add-on maintainers may want to lay claim to a single "add-on ID".
+It is important for users to expect consistent authorship for a single add-on ID for trust and security reasons.
+It would be misleading and potentially risky to be encouraged to update to a fork or alternative add-on without permission from the original maintainers.
+See [the submission guide](../submitters/submissionGuide.md#approval-process) for further reasoning behind this.
+This registration process ensures a specific add-on ID is only submitted by the same group of authorised maintainers.
+
+#### Process
+The process for reviewing pending first submissions is as follows:
+
+1. A Pull Request named "Add `GitHubName` as an approved submitter for `addonId`" will be waiting on review.
+   - Example: https://github.com/nvaccess/addon-datastore/pull/2674
+1. Open the referenced issue
+1. Check the source code link in the referenced issue.
```

Comment:
```suggestion
1. Check the source code link in the referenced issue, and the download URL.
```

Reply:
"The source code" mabe a confusing expression. It seems to be related to the source URL, but a submitter may provide a source URL not related to the download URL.

Reply:
To be clear, we are only checking the source code link, not the download URL.
We are checking for clear ownership over the add-on, i.e. a project or clear form of ownership for the add-on distribution, not performing a code review.

Reply:
VirusTotal and CodeQL scanning will be the limitation of any form of code review.